README.md

```diff
@@ -125,6 +103,46 @@ rokuDeploy.publish({
 });
 ```
 
+Can't find what you need? Here are all of the public functions:
+- stage
+- zip
+- sideload
+- getFilPaths
+- keyPress
+- keyUp
+- keyDown
+- sendText
+- closeChannel
+- rekeyDevice
+- createSignedPackage
+- deleteDevChannel
+- captureScreenshot
+- getOptions
+- checkRequiredOptions
+- getDeviceInfo
+- getDevId
+
+And here are all the possible cli commands:
+- bundle
+- deploy
+- package
+- exec
+- keypress
+- keyup
+- keydown
+- sendText
+- stage
+- sideload
+- squash
+- rekey
+- createSignedPackage
+- deleteDevChannel
+- screenshot
+- getDeviceInfo
+- getDevId
+- zip
+
+
```

Comment:
Remove the "you can provide a callback"  stuff, we scrapped that

Reply:
Done